### PR TITLE
MM-11294 Update to LDAP button help text

### PR DESCRIPTION
**Pull request by wiersgallak**
Updated help text for AD/LDAP Test button & AD/LDAP Synchronize Now button

Please make sure you've read the [pull request](http://docs.mattermost.com/developer/contribution-guide.html#preparing-a-pull-request) section of our [code contribution guidelines](http://docs.mattermost.com/developer/contribution-guide.html).

When filling in a section please remove the help text and the above text.

#### Summary
Updates the help text for AD/LDAP Test button & AD/LDAP Synchronize Now button

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-11294

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] Ran `make check-style` to check for style errors (required for all pull requests)
- [ ] Ran `make test` to ensure unit and component tests passed
- [ ] Added or updated unit tests (required for all new features)
- [ ] Has server changes (please link)
- [ ] Has redux changes (please link)
- [x] Has UI changes
- [x] Includes text changes and localization file ([.../i18n/en.json](https://github.com/mattermost/mattermost-webapp/blob/master/i18n/en.json)) updates
- [ ] Touches critical sections of the codebase (auth, posting, etc.)

### CODE REVIEW

File: components/admin_console/admin_definition.jsx [hunk 1]
```diff
@@ -922,7 +922,7 @@ export default {
                             label: 'admin.ldap.sync_button',
                             label_default: 'AD/LDAP Synchronize Now',
                             help_text: 'admin.ldap.testHelpText',
-                            help_text_default: 'Initiates an AD/LDAP synchronization immediately.',
+                            help_text_default: 'Initiates an AD/LDAP synchronization immediately and provides details on the status of the synchronization in the table below. This table also shows the status of synchronization executed based on the Synchronization Interval setting. Please review log file and [documentation](!https://docs.mattermost.com/deployment/sso-ldap.html) to troubleshoot errors.',
```
Comment (jasonblais) [inline components/admin_console/admin_definition.jsx]: Let's link to a mattermost.com page, which in turn redirects to docs. That way if the docs page moves/changes, we just have to fix the redirect from mattermost.com without product changes.

File: components/admin_console/admin_definition.jsx [hunk 2]
```diff
@@ -922,7 +922,7 @@ export default {
                             label: 'admin.ldap.sync_button',
                             label_default: 'AD/LDAP Synchronize Now',
                             help_text: 'admin.ldap.testHelpText',
```
[the remark this answers is not in the document]
Comment (wiersgallak) [reply components/admin_console/admin_definition.jsx]: Bug ticket created https://mattermost.atlassian.net/browse/MM-11666

**Comment by wiersgallak** [comment]
Thanks @jasonblais .  I've made the edits you suggested and have asked Rich to help update the redirect link.

**Comment by wiersgallak** [comment]
Closing this PR - Changes will be made in another.

**Pull request closed without merging**